import.html

```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
                         <br/><select id="tileServer">
                             <option value="bing">bing</option>
                             <option value="digital_globe">digital_globe</option>
-                            <option value="google">google</option>
```

Comment:
What's the reason for removing Google here? Should we just remove the line outright rather than commenting it out?